src/reducers/index.js

```diff
@@ -13,9 +13,17 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+    return {
+             state,
```

Comment:
It would be best to not directly mutate the state so use spread operator for that.